openedx-wasm/untrusted_code/run.py

```diff
@@ -0,0 +1,45 @@
+import wasm_helper
+
+globals().update(wasm_helper.read_globals_from_file())
+
+
+import random
+try:
+	randomseed = int(anonymous_student_id, 16)
+except:
+	randomseed = 1
+
+random.seed(randomseed)
+
+ar_a = random.randrange(225000,275000, step=5000)
+ar_b = random.randrange(125000,150000, step=1000)
+pct_a = round(random.uniform(0.01,0.03),2)
+pct_b = round(random.uniform(0.08,0.12),2)
+beg_ada = random.randrange(14000,16000, step=10)
+end_ada = (ar_a * pct_a) + (ar_b * pct_b)
+writeoffs = random.randrange(3800,5400, step=100)
+bde = round(end_ada - beg_ada + writeoffs,2)
+
+def stringify(var, usd=True):
+	if usd == False:
+		var_str = "{:,}".format(var)
+		return var_str
+	if var >= 0:   
+		var_str = "${:,}".format(var)
+	else:
+		var = -var
+		var_str = "-" + "${:,}".format(var)
+	return var_str
+
+ar_a_str = stringify(ar_a)
+ar_b_str = stringify(ar_b)  
+pct_a_str = stringify(pct_a*100, False)
+pct_b_str = stringify(pct_b*100, False)
+beg_ada_str = stringify(beg_ada)
+end_ada_str = stringify(end_ada)
+writeoffs_str = stringify(writeoffs)   
+bde_str = stringify(bde)
+
+
+
+wasm_helper.write_globals_to_file(globals())
```

Comment:
Seems like this file or perhaps everything in `untrusted_code` should be .gitignored?